pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -331,6 +363,8 @@ func (f *framework) RunScorePlugins(state *CycleState, pod *v1.Pod, nodes []*v1.
 		return nil, NewStatus(Error, msg)
 	}
 
+	klog.V(10).Infof("%+v", pluginToNodeScores)
```

Comment:
nit: please add "pluginToNodeScores after Score:%+v", and similarly "pluginToNodeScores after NormalizeScore:%+v", "pluginToNodeScores after ApplyWeights:%+v"

Reply:
done